common/database.cpp

```diff
@@ -925,6 +925,22 @@ void Database::GetCharName(uint32 char_id, char* name) {
 	}
 }
 
+const char* Database::GetCharNameByID(uint32 char_id) {
+	std::string query = fmt::format("SELECT `name` FROM `character_data` WHERE id = '{}'", char_id);
```

Comment:
`id` does not need quotes around it because its an integer, query plan in the optimizer does change when things are quoted

Reply:
Word, saw examples using it and some without, will fix. 👍